java/packagescanner.go

```diff
@@ -34,12 +34,14 @@ var (
 )
 
 const DefaultSearchAPI = `https://search.maven.org/solrsearch/select`
+const DefaultRequestTimeout = 10
```

Comment:
If this is configurable, I'd like the default timeout to be much smaller, like 2 seconds.